values-secret.yaml.template

```diff
@@ -3,4 +3,25 @@ secrets:
   # NEVER COMMIT THESE VALUES TO GIT
   config-demo:
     # Secret used for demonstrating vault storage, external secrets, and ACM distribution
-    secret: PLAINTEXT
+    secret: VALUE
+
+  # Required for automated spoke deployment
+  aws:
+      access_key_id: VALUE
+      secret_access_key: VALUE
+
+# Required for automated spoke deployment
+# DO NOT use ~/ , it will appear to work be the secrets will be empty
+files:
+  # ssh-rsa AAA...
+  publickey: ~/.ssh/id_rsa.pub
```

Comment:
I think we should leave them commented out? Won't 'load-secrets' explode if the files do not exist?